### ISSUE TITLE
96888019 Download multiple images (Erica Ford)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#1040]
Issue reported by **Erica Ford** at 2018-01-26 09:20:19 UTC
Time: 2018-01-26T09:15:37+00:00
Session ID: 298ef7f133259351782cde2942b0e9b8
Problem Page URL: []()
Previous Page URL: [https://www.freereg.org.uk/contacts/new](https://www.freereg.org.uk/contacts/new)
Reported Issue:
It would be really useful if we could download images in batches (or even download all - heaven) rather than having to click and download each image separately - takes a very long time when you've been allocated 80 files! (you may have thought of this but I couldn't find how so I'm guessing you haven't done it?)    

![Screenshot](http://www.freereg.org.uk/uploads/contact/screenshot/5a6af2d3f493fdb03fba0e1e/_old_)

**Comment by SteveBiggs** [comment]
Could this be incorporated in the manage images work to download multiple images?

**Comment by edickens** [comment]
Have you tried clicking the first in the range, then Shift+Click for the last in the range?  Or Shift+Click to select a number of individual images?  This works on the search when selecting counties and places.

**Comment by SteveBiggs** [comment]
Eric, on which view are you suggesting to do this?

**Comment by Captainkirkdawson** [comment]
It would be nice to know who and what is the requirement that leads to the desire for downloading so many pages at once. A transcriber can only transcribe 1 page at a time. Coordinators can quickly view pages on line. Many of our "suppliers" have concerns over multiple copies being made by multiple people.

**Comment by edickens** [comment]
It is easier to transcribe by setting up a split screen and you need the image on your computer.  Also, you need to go back at times to double check or to compare writing.
But the transcriber should never have more than one image group at a time and never be assigned a complete set of images.
FamilySearch has a system where you save the images on your computer, but they are password protected and when you get your next assignment the previous one is deleted.  You have to use their software.  However, this is not a good system as the transcriber needs to get to know the handwriting in the register and look at a large range of images, and not be given ten random ones at a time.

**Comment by SteveBiggs** [comment]
Kirk, this was a request by a transcriber and I have had a few others ask this from time to time. I agree that we don't want to encourage too many images to be held on their PC and reiterate that they must delete when completed. However, there are times when it can be useful to download a number of images, for example, I've been on long haul flights that had no internet access so downloaded  several images to give me something to do on the journey and there are other reasons why one might be offline for a period and want to do some transcribing, but I wouldn't say it's a high priority issue.

**Comment by edickens** [comment]
Steve asked "Which view".  I was wrong.  It is the Upload that has a pick-list.
So I suggest we need an Action at IG level to "Download images for transcribing" which will have a pick-list of images in that IG, but also have a warning to say that they should not keep them on their computer after transcribing is completed and checked.  I'm not sure, but you may want to add the statement that this would then be termed as copying, which is against T&C.  When they indicate "Completed" there could be a message to ask if they have deleted the images from their computer and they should confirm with a "Yes".  But that gets a bit complicated as they could be their images and they may not have downloaded the images.

**Comment by Sherlock21** [comment]
I would  say that its Essential to be able to log on to wherever one is to get a set of images to transcribe, and download a couple of dozen images in one go, whilst at that site, so i have them on my computer all in one set. Then i can have a quick look through the lot, and get the feel of which are the good pages and which the bad.  And then plan my working sessions accordingly. Its no good having to download one image, and transcribe that, before being able to get the next one as has been said above.  Because its just as useful to be about to look FORWARD as Backwards, to understand hard- to- read handwriting. And sometimes - e.g. with images of Marriages, there are only 2 records on a page in later times, where I can have Transcribed half a dozen pages, in the time taken to log on and download one image page.  So doing the download in bundles is the time efficient way to do the job ( i.e. in the manner of a block, not going round a loop of one at a time whilst still logged on as one had to do in F1).

**Comment by Sherlock21** [comment]
I agree with all of Eric D's points in his note 27 Jan above.
Also, a  Transcriber will develop their own way of working, and if other ways are forced upon them which they dope like, they won't do it. - and i speak from both sides of the fence here.

**Comment by Captainkirkdawson** [comment]
I disagree totally with the suggestion by EricD "So I suggest we need an Action at IG level to "Download images for transcribing" which will have a pick-list of images in that IG"
The transcriber should  only be permitted to download images that are assigned to them. It Is likely possible that we can make the download of images from an assignment be a multiple select. Will discuss this with Bev on my return. It is not as easy as the multiple select on upload because of the need for communication between 2 servers which is why the current implementation is 1 at a time.
I do like the idea of reminding the transcriber that they should delete the images when the SC has agreed that the assignment is complete.

**Comment by edickens** [comment]
Yes, it has to only be images assigned to the transcriber.  Forgot to mention that.  And I can see now that it may not be easy to do.
